### PR TITLE
Mostr docker postgres

### PR DESCRIPTION
**Pull request by Maryl12342**
Hello, please review my changes. 
Regarding removing lines with .env in docker-compose- I did it because I got following error:
"ERROR: Couldn't find env file: *****\Documents\GIT\.env"

### CODE REVIEW

**Comment by mgierm** [comment]
> Hello, please review my changes. Regarding removing lines with .env in docker-compose- I did it because I got following error: "ERROR: Couldn't find env file: *****\Documents\GIT.env"

Please copy [.example.env](https://github.com/klepper24/WarsawPublicTransport/blob/main/.example.env) content into `.env` file which you should create in the project root directory (next to `.example.env`). Let me know if it helps or not.

**Comment by Maryl12342** [comment]
@mgierm it worked out, thanks :) I restored lines with .env in docker-compose.

File: postgresdb/WarsawTransportState.sql [hunk 1]
```diff
@@ -0,0 +1,61 @@
+CREATE SCHEMA "dbo";
+
+CREATE TABLE "WarsawTransportState"."dbo"."stops" (
+  "id" int PRIMARY KEY,
+  "full_name" int,
```
Comment (mgierm) [inline postgresdb/WarsawTransportState.sql]: Just to double-check - is it really `int`?

File: docker-compose.yml [hunk 2]
```diff
@@ -118,7 +118,7 @@ services:
       airflow-init:
         condition: service_completed_successfully
     networks:
-      - internal-network
+      - internal-network
```
Comment (mgierm) [inline docker-compose.yml]:
```suggestion
      - internal-network
```

File: docker-compose.yml [hunk 3]
```diff
@@ -351,7 +351,25 @@ services:
...
+    ports:
+      - 5432:5432
+    volumes:
+      - ./postgresdb/database:/bitnami/postgresql
+      - ./postgresdb/WarsawTransportState.sql:/docker-entrypoint-initdb.d/WarsawTransportState.sql
+
+    restart: always
+    networks:
+        - internal-network
+
```
Comment (mgierm) [inline docker-compose.yml]:
```suggestion
```

**Review by mgierm — COMMENTED** [review]
Looks good overall :+1: Just a few little comments.

**Comment by Maryl12342** [comment]
@meg25ellele @klepper24  @mgierm, I uploaded files with latest db schema and with minor changes in docker-compose file. Please take a look :)

File: docker-compose.yml [hunk 4]
```diff
@@ -351,8 +351,8 @@ services:
       - ./processor/scripts:/opt/spark/execution_scripts
     restart: always
     networks:
-        - internal-network
-       
+        - internal-network
```
Comment (mgierm) [inline docker-compose.yml]:
```suggestion
        - internal-network
```

File: docker-compose.yml [hunk 5]
```diff
@@ -369,7 +369,7 @@ services:
     restart: always
     networks:
         - internal-network
-    
+
```
Comment (mgierm) [inline docker-compose.yml:372]:
```suggestion
```

File: postgresdb/WarsawTransportState.sql [hunk 6]
```diff
@@ -2,13 +2,14 @@ CREATE SCHEMA "dbo";
...
-  "longitude" double precision,
-  "latitude" double precision,
+  "full_name" varchar,
+  "stop_longitude" double precision,
+  "stop_latitude" double precision,
   "street" varchar,
   "unit" varchar,
   "post" varchar,
-  "is_depot" int
+  "is_depot" int,
```
Comment (mgierm) [inline postgresdb/WarsawTransportState.sql]: It's probably the result of the mistake in the diagram - but I believe it should be `boolean` instead `int`

**Review by mgierm — CHANGES_REQUESTED** [review]

**Comment by Maryl12342** [comment]
Changes added.

**Review by klepper24 — APPROVED** [review]
cool :)

**Pull request merged**